client/app/components/packages/landuse-form/zoning-resolution-dropdown.js

```diff
@@ -1,12 +1,20 @@
 import Component from '@glimmer/component';
+import { action } from '@ember/object';
 import { tracked } from '@glimmer/tracking';
 
 export default class PackagesLanduseFormZoningResolutionComponent extends Component {
-  @tracked chosenZoningResolution;
+  @tracked chosenZoningResolution = this.args.landuseActionForm.data.zoningResolution || null;
```

Comment:
We can probably remove `searchPlaceholder` computed property and just put raw text in the placeholder field on the template if we have this right?